languages/bash.scm

```diff
@@ -5,15 +5,17 @@
 (string) @leaf
 
 ; Allow blank line before
-; TODO Add directives here...
 [
   (comment)
   (if_statement)
+  (command)
+  (list)
+  (pipeline)
+  ; TODO: etc.
 ] @allow_blank_line_before
 
 ; Prepend spaces
 ; (Not "surround", to avoid extraneous spaces between delimiters)
```

Comment:
I think we can surround, and let the post-processing take care of extraneous spaces

Reply:
The reason this is in here is to protect the `<CLAUSE>; <CLAUSE>` case; i.e., no space before the delimiter. I don't believe the post-processing deals with this.

That said, I've fixed this in #174 by making the "surround with spaces" list much more restrictive.